src/Chapter18/Listing18.18.UpdatingCommandLineHandler.TryParseToHandleAliases.cs

```diff
@@ -85,16 +94,16 @@ public class CommandLineHandler
                 else
                 {
                     success = false;
-                    errorMessage = string.Format(
-                        "Data type '{0}' on {1} is not supported.",
-                        property.PropertyType.ToString(),
-                        commandLine.GetType().ToString());
+                    errorMessage = 
+                        $@"Data type '{ property.PropertyType.ToString() }' on {
+                        commandLine.GetType().ToString() } is not supported.";
                 }
             }
             return success;
         }
+        #endregion INCLUDE
 
-// Justification: Not fully implemented.
+        // Justification: Not fully implemented.
```

Comment:
```suggestion
// Justification: Not fully implemented.
```
nit: align the comment with the #pragma warning disable